Dockerfile

```diff
@@ -12,8 +12,7 @@ WORKDIR /app
 
 RUN \
     make virtualenv && \
-    .venv/bin/pip install --constraint jobs/requirements.txt jobs/ && \
-    .venv/bin/pip freeze | grep -v -- 'buildhub' > dependencies.txt
+    .venv/bin/pip install -r jobs/requirements/default.txt -c jobs/requirements/constraints.txt
```

Comment:
I still don't really get this line. What's the point?

If the point is to make a virtualenv *only* for the benefit on `make lambda.zip` then why can't this be run as part of that? Only.

Reply:
To be honest I don't remember why we generate this `dependencies.txt` file.